dxp/sennajs_integration.md

```diff
@@ -84,30 +84,30 @@ By default, Senna.js dispatches some events during the navigation process that c
 -   `beforeNavigate`
     Dispatched before navigation starts
 
+-   `startNavigate`
+    Fired when navigation begins
+
 -   `beforeScreenFlip`
     Dispatched before a screen is visible
 
--   `endNavigate`
-    Dispatched after the content has been retrieved and inserted onto the page
+-   `screenLoad`
```

Comment:
Is it true that `beforeScreenFlip` comes before `screenLoad`?

Reply:
no, it's not. I fixed that in https://github.com/liferay/liferay-frontend-guidelines/pull/223/commits/2e2b34b5b3726a1182a4a1b16801bf2977b29771